### ISSUE TITLE
Public goods balancer pool

### ISSUE THREAD

**Issue by chad-fi** [Giveth/GIVeconomy#628]

**Comment by chad-fi** [comment]
Fabio, dev at BAL would know who to talk to.

**Comment by chad-fi** [comment]
Update from Freiheit:  asked luukdao from primedao/kolectivo if he has any suggestion or direct contact. It'd be great to have a direct contact with Balancer as they are planning to launch on Gnosis Chain, so they could be interested in further cooperations with Giveth (like BAL rewards on Gnosis).

**Comment by chad-fi** [comment]
Lauren to share DM chadfi with Fabio's handle - discussion to start this week.

**Comment by chad-fi** [comment]
Balancer hasn't done many token swaps recently b/c of the veBAL demand. Fabio said they would likely be down to help facilitate a public goods pool a la this tweet:

https://twitter.com/mZargham/status/1474436731516637188

**Comment by chad-fi** [comment]
I would like to proceed with a public goods pool, and in parallel request that Balancer do a BAL<>GIV swap to align us both as partners. I'll word smith something after we get a soft consensus here on how we want to proceed (i.e. BAL<>GIV pool only, PG pool only, PG pool + GIV<>BAL pool)

**Comment by chad-fi** [comment]
Other public good DAOs:
1Hive
Shapeshift
Gitcoin (probably won't do it - can't do LPs)
DappNode
BrightID
Opolis

**Comment by chad-fi** [comment]
re: Opolis
Opolis may be inflationary and drag down GIV price, this would be a non-starter. Perhaps more research into their inflation schedule and token unlock

**Comment by chad-fi** [comment]
Panvala is a no-go, see #631

**Comment by chad-fi** [comment]
https://forum.giveth.io/t/balancer-public-goods-pool/733

**Comment by fr3iheit** [comment]
reached out RAD community in their forum:
https://radicle.community/t/shall-rad-join-the-forces-to-create-a-balancer-public-goods-pool/3005

reached out @Pol_Lanski for Dappnode

**Comment by chad-fi** [comment]
chadfi pinged Optimism contact
chadfi posted in Balancer Discord #partnerships channel

**Comment by chad-fi** [comment]
Follow up tweet with Medium article if engagement is adequate.

Ping GIV community for Optimism contacts (two-fold, one for BAL pool, one for possible grant).

**Comment by laurenluz** [comment]
Talked to Griff about this. He thinks its a good idea, but we don't have the bandwidth to manage it right now. haulting comms and putting this in the icebox.
